modules/nf-core/topas/gencons/main.nf

```diff
@@ -0,0 +1,45 @@
+process TOPAS_GENCONS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::topas=1.0.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/topas:1.0.1--hdfd78af_1':
+        'biocontainers/topas:1.0.1--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(vcf)
+    tuple val(meta), path(reference)
```

Comment:
missing indels VCF file

```
topas gencons -?
TOPAS - TOolkit for Processing and Annotation of Sequence data
-snps                   <snp.vcf>           [REQUIRED] input snps.vcf file
-indels                 <indels.vcf>        [OPTIONAL] input indels.vcf file
-o                      <output file>       [REQUIRED] the output path of the consensus fasta file
-ref                    <ref.fasta>         [REQUIRED] the refernce fasta file
-fai                    <reference.fna.fai> [OPTIONAL] The path to the fasta index of the reference genome in order tothe right number of trailing 'N' to the consensus sequence.
-consensus_ratio        <double>            [OPTIONAL] The minimum percentage of reads of a base in relation to all other bases, so that this base is called for the consensus sequence. DEFAULT: 0.75.
-major_allele_coverage  <integer>           [OPTIONAL] The minimum number of reads that are required declaring a base as a part of the consensus sequence. DEFAULT: 5.
-total_coverage         <integer>           [OPTIONAL] The minimum total number of reads that are required making a consensus call of bases in the vcf file. DEFAULT: Not Set.
-punishment_ratio       <double>            [OPTIONAL] The ratio which the coverage of a base exchange from C/G to T/A is multiplied with. DEFAULT: 0.8.
-name                   <string>            [OPTIONAL] A name (e.g. of sample) appearing in all FASTA headers.
-suppress_warn                              [OPTIONAL] If this option is set, all warnings are suppressed.
-vcf_out                                    [OPTIONAL] Additional output in vcf format.
-minor                                      [OPTIONAL] If this option is set, only the minor 
```

Reply:
added